src/stream-ciphers.rst

```diff
@@ -333,19 +333,19 @@ information, the server that handles it encrypts it using a strong block
 cipher in :term:`CBC mode` with a fixed key. For now, we'll assume that that
 server is secure, and there's no way to get it to leak the key.
 
-Mallory gets a hold of all of the rows in the database. Perhaps she did
+Mallory gains access to all rows in the database. Perhaps she did
 it through a SQL injection attack, or maybe with a little social
 engineering. [#]_ Everything is supposed to remain secure: Mallory only
-has the ciphertexts, but she doesn't have the secret key.
+has the ciphertexts, but not the secret key.
 
 .. [#]
-   Social engineering means tricking people into things they shouldn't
-   be doing, like giving out secret keys, or performing certain
-   operations. It's usually the most effective way to break otherwise
+   Social engineering means tricking people into doing things they should not
+   be doing. For example, giving out secret keys or performing certain
+   operations. It is usually the most effective way to break
    secure cryptosystems.
 
 Mallory wants to figure out what Alice's record says. For simplicity's
-sake, let's say there's only one ciphertext block. That means Alice's
+sake, lets say there is only one ciphertext block. This means Alice's
```

Comment:
The original `let's say` appears correct?

Reply:
Reverted